### PR TITLE
Add more options

### PR DESCRIPTION
**Pull request by nekhtiari**

### CODE REVIEW

File: blocks/snap-polarimetric/UP42Manifest.json [hunk 1]
```diff
@@ -22,9 +22,12 @@
         "type": "string",
         "enum": ["VV", "VH"]
       }},
+    "calibration_band": {"type": "array", "default": ["sigma"]},
```
Comment (markusuwe) [inline blocks/snap-polarimetric/UP42Manifest.json]: I think this should rather be "calibration_method" but If encourage you to research how this is normally called. I never heard it called "band" before. Or did the term come from SNAP itself?

File: blocks/snap-polarimetric/UP42Manifest.json [hunk 2]
```diff
@@ -22,9 +22,12 @@
...
+    "specklefilter": {"type": "boolean", "default":  true},
     "mask": {"type":  "array", "default":  null, "items": {"type": "string", "enum": ["land", "sea"]}},
     "tcorrection": {"type":  "boolean", "default":  true},
-    "clip_to_aoi": {"type":  "boolean", "default":  false}
+    "clip_to_aoi": {"type":  "boolean", "default":  false},
+    "lintodb": {"type": "boolean", "default": true}
```
Comment (markusuwe) [inline blocks/snap-polarimetric/UP42Manifest.json]: Please make parameter naming more consistent. Generally it appears we use underscore to separate terms. Worthwhile going through all the parameter names.

Comment (markusuwe) [inline blocks/snap-polarimetric/UP42Manifest.json]: Capabilities need to be updated, if no log-transform is applied the output will be float, not int.

File: blocks/snap-polarimetric/src/template/snap_polarimetry_graph.xml [hunk 3]
```diff
@@ -22,9 +22,9 @@
       <createGammaBand>false</createGammaBand>
       <createBetaBand>false</createBetaBand>
       <selectedPolarisations>${upcase_polarisation}</selectedPolarisations>
-      <outputSigmaBand>true</outputSigmaBand>
-      <outputGammaBand>false</outputGammaBand>
-      <outputBetaBand>false</outputBetaBand>
+      <outputSigmaBand>${sigma_band}</outputSigmaBand>
```
Comment (markusuwe) [inline blocks/snap-polarimetric/src/template/snap_polarimetry_graph.xml]: Hmmm maybe I am wrong about what I said above and this is a common naming convention...

File: blocks/snap-polarimetric/tests/test_snap_polarimetry.py [hunk 4]
```diff
@@ -565,3 +566,17 @@ def test_run_scene(safe_file):
         shutil.rmtree("/tmp/output/")
     if os.path.exists("/tmp/input/data.json"):
         os.remove("/tmp/input/data.json")
+
```
Comment (markusuwe) [inline blocks/snap-polarimetric/tests/test_snap_polarimetry.py]: Please add more test cases. With all the options now available, we should have more tests.

File: blocks/snap-polarimetric/UP42Manifest.json (hunk 2 above)
Comment (nekhtiari) [reply blocks/snap-polarimetric/UP42Manifest.json]: so, in this case, I should use "or"?

Comment (markusuwe) [reply blocks/snap-polarimetric/UP42Manifest.json]: Yup, sounds right.

**Pull request merged**